### DISCUSSION TITLE
Version 1.2 Release Candidate 1

### DISCUSSION THREAD

**Discussion by ErinaSugino** [ErinaSugino/Starbound-SBR-Mod#92] [Update Builds]
## Version 1.2 Patchnotes
- Persistent crewmember data
- Player arousal effects and "in heat" mechanics
- NPC sub-genders
- Actor Relayering (with additional layers for ears and tails and offsets per species)
- Complete rework of sextalk
- Fixes for common issues like Anom's Outpost Overhaul, Vulpes Race, Cutebound Avians and forever-kids

## RC1 Notes
- Exhaustive testing needed before release.
- Known missing things from this build: Sextalk for cowgirl and toys; potentially more efficient sextalk allocation.

### Important Notes
- Due to the complete overhaul, every sextalk altering mod is incompatible now.

<hr /><em>This discussion was created from the release <a href='https://github.com/ErinaSugino/Starbound-SBR-Mod/releases/tag/v1.2_rc1'>Version 1.2 Release Candidate 1</a>.</em>

**Comment by luciodark** [comment]
Player arousal and heat mechanics don't occur despite being enabled and set to values that should make them proc almost immediately 
[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/14979812/starbound.log)
